src/backend/shortcuts/shortcuts/shortcuts.ts

```diff
@@ -139,4 +163,105 @@ function shortcutFiles(gameTitle: string) {
   return [desktopFile, menuFile]
 }
 
+async function generateMacOsApp(gameInfo: GameInfo) {
+  const { title, app_name } = gameInfo
+
+  logInfo('Generating macOS shortcut', { prefix: LogPrefix.Backend })
+
+  const appShortcut = join(userHome, 'Applications', `${title}.app`)
+  const macOSFolder = `${appShortcut}/Contents/MacOS`
+  const resourcesFolder = `${appShortcut}/Contents/Resources`
+  const plistFile = `${appShortcut}/Contents/Info.plist`
+
+  // create the .app folder
+  if (!existsSync(appShortcut)) {
+    mkdirSync(appShortcut, { recursive: true })
+  }
+
+  if (!existsSync(resourcesFolder)) {
+    mkdirSync(resourcesFolder, { recursive: true })
+  }
+
+  // convert the icon to icns
+  const isIconIcns = await convertPngToICNS(app_name, gameInfo, resourcesFolder)
+
+  // if the icon was converted, we can continue
+  if (isIconIcns) {
+    const plist = `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Some documentation link on the structure of this might be helpful for the future

Reply:
The documentation is basically copy paste from a steam shortcut.
Looks pretty generic to me. Mac uses those plist files to all types of configs.